ddp-workspace/projects/ddp-pancan/src/assets/i18n/es.json

```diff
@@ -23,340 +23,579 @@
-                "LearnMoreAboutUsButton": "Learn more about us"
+                "Title": "[ES]Every patient's experience with cancer should be counted.",
+                "Text": "[ES]Patients’ medical records, tumor samples, and personal experiences hold clues—about which cancers will respond or become resistant to which therapies—that often can’t be found in research labs or clinical trials. <span class=\"italic\">Count Me In</span> enables cancer patients anywhere to share their information with researchers everywhere, to help reveal patterns in the data.",
+                "LearnMoreAboutUsButton": "[ES]Learn more about us"
             },
             "JoinSection": {
-                "Title": "Who can join?",
-                "Text1": "Anyone who has ever been diagnosed with cancer may participate if they live in the USA or Canada.",
-                "Text2": "We welcome people of all ages, genders, races, ethnicities, and abilities."
+                "Title": "[ES]Who can join?",
+                "Text1": "[ES]Anyone who has ever been diagnosed with cancer may participate if they live in the USA or Canada.",
+                "Text2": "[ES]We welcome people of all ages, genders, races, ethnicities, and abilities."
             },
             "ParticipateSection": {
-                "Title": "Here’s how to participate",
+                "Title": "[ES]Here’s how to participate",
                 "Step1": {
-                    "Title": "STEP 1",
-                    "Time": "10-15 minutes",
-                    "Description": "Provide consent and tell us where you’ve been treated"
+                    "Title": "PASO 1",
+                    "Time": "[ES]10-15 minutes",
+                    "Description": "[ES]Provide consent and tell us where you’ve been treated"
                 },
                 "Step2": {
-                    "Title": "STEP 2",
-                    "Time": "5 minutes",
-                    "Description": "Answer questions about your cancer"
+                    "Title": "PASO 2",
+                    "Time": "[ES]5 minutes",
+                    "Description": "[ES]Answer questions about your cancer"
                 },
                 "Step3": {
-                    "Title": "STEP 3",
-                    "Time": "10 minutes",
-                    "Description": "Provide samples (we’ll send a kit)"
+                    "Title": "PASO 3",
+                    "Time": "[ES]10 minutes",
+                    "Description": "[ES]Provide samples (we’ll send a kit)"
                 },
                 "Step4": {
-                    "Title": "STEP 4",
-                    "Description": "Learn with us along the way"
+                    "Title": "PASO 4",
+                    "Description": "[ES]Learn with us along the way"
```

Comment:
suggest 
`"Aprenda junto con nosotros"` 
( got it from ddp-mbc/es.json, see ThirdStep)